.release-notes/3675.md

```diff
@@ -0,0 +1,7 @@
+## Fix memory safety problem with Array.from_cpointer
+
+Previously, the `from_cpointer` method in arrays would trust the user to pass a
```

Comment:
For future reference, in general for markdown in ponylang projects, we do breaking lines only for paragraphs etc. If you have time to update this, that would be appreciated but it's not of huge importance in the release notes.

Reply:
Thanks for letting me know, should be fixed now